The-Chonky-Panda/Atelier
# Individual review tile update

## Description

Made some style updates to the individual review tile.
Also integrated react-router from dev, and update test setting for that.
Also added imports for icons to eliminate console warnings for tests.

## How to test

`npm test`
Manual test:
Should look like this: 
<img width="735" alt="Screen Shot 2021-09-21 at 10 48 58 PM" src="https://user-images.githubusercontent.com/43324065/134283092-f3908184-8769-4e23-b168-9c9953fdad7b.png">

1. click on `show more` would show the remaining text if any
2. display review photos if any (test product id: 47422)
3. click on a review photo would pop up a modal showing that photo in full resolution
4. change product id in url should update reviews

## Notes


## Issue tracking

https://trello.com/c/2DNK0LT6
